app/models/dialog/ansible_playbook_service_dialog.rb

```diff
@@ -8,7 +8,7 @@ def self.create_dialog(label, extra_vars, hosts = 'localhost')
     # The job_template contains the playbook
     def create_dialog(label, extra_vars, hosts = 'localhost')
       Dialog.new(:label => label, :buttons => "submit,cancel").tap do |dialog|
-        tab = dialog.dialog_tabs.build(:display => "edit", :label => "Basic Information", :position => 0)
+        tab = dialog.dialog_tabs.build(:dialog => dialog, :display => "edit", :label => "Basic Information", :position => 0)
```

Comment:
I don't think you need to include the `:dialog => dialog` piece here, because we're doing `dialog.dialog_tabs.build`. I tried it in console without explicitly passing in the dialog and `tab.dialog` ends up with a reference to the dialog as expected. Is there another reason for passing in the dialog explicitly here (and everywhere else this has been changed)?